bgpd/bgp_evpn.c

```diff
@@ -4333,6 +4333,60 @@ static void update_autort_vni(struct hash_bucket *bucket, struct bgp *bgp)
 	}
 }
 
+/*
+ * Handle autort change for L3VNI.
+ */
+static void update_autort_l3vni(struct bgp *bgp)
+{
+	if (!bgp->l3vni)
```

Comment:
You already have this check calling inside `bgp_evpn_handle_autort_change()`. So, I think that test is not necessary.

Reply:
ok. change done.